lib/newsletter_exclusion.rb

```diff
@@ -0,0 +1,93 @@
+      'disability-and-sickness-benefits-check-your-entitlements',
+      'personal-independence-payment-an-introduction',
+      'universal-credit-for-disabled-people',
+      'financial-support-if-you-or-your-child-has-a-disability',
+      'how-to-appeal-against-a-disability-benefits-decision',
+      'funding-to-adapt-your-home-for-accessibility',
+      'charitable-grants-for-ill-or-disabled-people',
+      'shopping-around-for-disability-aids-and-equipment',
+      'motability-blue-badge-scheme-and-discounted-travel',
+      'money-saving-tips-and-discounts-for-disabled-people',
+      'buying-insurance-if-youre-ill-or-disabled',
+      'getting-a-mortgage-if-youre-ill-or-disabled',
+      'getting-a-loan-if-you-are-ill-or-disabled',
+      'early-retirement-because-of-illness-or-disability',
+      'long-term-care-services-at-a-glance',
+      'planning-ahead-for-when-you-cant-manage-your-money',
+      'care-services-to-help-you-stay-in-your-own-home',
+      'preparing-for-illness-old-age-and-death',
+      'care-home-or-home-care',
+      'choosing-the-right-care-home',
+      'what-if-youre-unhappy-with-the-care-you-receive',
+      'how-to-challenge-your-local-authority-over-your-care',
+      'solving-problems-when-family-members-are-helping-with-your-money',
+      'using-a-personal-assistant-to-provide-your-care',
+      'when-does-someone-need-formal-help-with-managing-their-money',
+      'setting-up-a-power-of-attorney',
+      'setting-up-a-trust',
+      'being-a-trustee',
+      'resolving-problems-with-attorneys-and-trustees',
+      'self-funding-your-long-term-care-your-options',
+      'are-you-eligible-for-nhs-continuing-care-funding',
+      'benefits-you-can-claim-when-you-have-care-needs',
+      'get-financial-advice-on-how-to-fund-your-long-term-care',
+      'local-authority-funding-for-care-costs-do-you-qualify',
+      'registered-nursing-care-contribution',
+      'means-tests-for-help-with-care-costs-how-they-work',
+      'how-a-local-authority-care-needs-assessment-works',
+      'direct-payments-explained',
+      'managing-direct-payments-to-pay-for-the-care-you-need',
+      'how-to-fund-your-long-term-care-a-beginners-guide',
+      'claiming-on-insurance-to-help-cover-the-costs-of-care',
+      'using-a-home-reversion-plan-to-pay-for-your-care',
+      'make-your-money-easier-to-manage-by-yourself',
+      'using-investment-bonds-to-pay-for-your-long-term-care',
+      'immediate-need-care-fee-payment-plans',
+      'using-a-lifetime-mortgage-to-pay-for-your-long-term-care',
+      'using-an-equity-release-scheme-to-fund-your-care',
+      'paying-your-own-care-costs-but-the-moneys-run-out',
+      'downsizing-your-home-to-fund-your-long-term-care',
+      'if-your-baby-is-stillborn',
+      'if-the-person-you-want-to-help-has-lost-mental-capacity',
+      'putting-your-affairs-in-order',
+      'preparing-for-illness-old-age-and-death',
+      'choosing-your-executor'
+    ]
+  end
```

Comment:
This is eventually (at some point later this week I believe!) going to be added as a flag against an article so this can be managed in the db/admin rather than as a hard coded list.